src/trio/_core/_io_kqueue.py

```diff
@@ -118,7 +118,9 @@ def current_kqueue(self) -> select.kqueue:
     @contextmanager
     @_public
     def monitor_kevent(
-        self, ident: int, filter: int
+        self,
+        ident: int,
+        filter: int,  # Name shadows builtin
```

Comment:
this comment gives zero info unless you know that it refers to A002, and that it's ignored in this file. So probably remove it, and/or add a longer comment saying that A002 is ignored in the file and why. But not sure either is warranted

Reply:
Changed in 67116bc